doc/source/data/inspecting-data.rst

```diff
@@ -151,46 +151,57 @@ Inspecting execution statistics
 Ray Data calculates statistics during execution for each operator, such as wall clock time and memory usage.
 
 To view stats about your :class:`Datasets <ray.data.Dataset>`, call :meth:`Dataset.stats() <ray.data.Dataset.stats>` on an executed dataset. The stats are also persisted under `/tmp/ray/session_*/logs/ray-data.log`.
+For more on how to read this output, see :ref:`Monitoring Your Workload with the Ray Data Dashboard <monitoring-your-workload>`.
 
 .. testcode::
     import ray
-    import time
+    import datasets
```

Comment:
for some reason i'm not seeing these import statements in the example code here? https://anyscale-ray--44093.com.readthedocs.build/en/44093/data/inspecting-data.html#inspecting-execution-statistics

Reply:
Hmm seems like it was from the missing space after the `.. testcode:: line`, fixed.